analytics.js

```diff
@@ -1307,4 +1317,6 @@ export {
   getUserTraits,
   getAnonymousId,
   setAnonymousId,
+  getGroupId,
+  getGroupTraits,
```

Comment:
Add these functions in the type declaration file also.

Reply:
Done. I missed it during the PR split.
I've added the missing `getUserTraits` API declaration as well.